README.md

```diff
@@ -54,7 +59,9 @@ Using Ruby...
 Write your code here...
 
 ```ruby
-# code here
+town[:residents].delete("Belle")
+town[:castle][:guest].push(town[:residents][1])
```

Comment:
👍 Both these lines are correct by themselves, but be careful running them in this order.

The first line removes "Belle" from index 1, leaving "Gaston" at index 1 now.
This means the second line actually adds "Gaston" to the castle guests instead of "Belle"!